### PR TITLE
Emailauthdel

### PR DESCRIPTION
**Pull request by kvdb**

### CODE REVIEW

File: docs/api/emailauth.md [hunk 1]
```diff
@@ -14,12 +14,6 @@ To create an emailauth for the current user:
 POST /user/emailauths HTTP/1.1
 ```
 
-To create an emailauth for another user:
-
-```http
-POST /users/:user/emailauths HTTP/1.1
```
Comment (joggienl) [inline docs/api/emailauth.md:20]: This does not work since you are deleting it?

Comment (kvdb) [reply docs/api/emailauth.md:20]: I couldn't find the url handler in the code that is supposed to do this, so that's why i deleted it.

Comment (kvdb) [reply docs/api/emailauth.md:20]: is/was that used in any its-tools script?

Comment (joggienl) [reply docs/api/emailauth.md:20]: not by my knowledge.. if I was to update it for another user, I would try impersonating first. So if you are sure, go ahead and merge 👍

**Pull request merged**